app/models/claim.rb

```diff
@@ -19,4 +19,11 @@ def self.fetch_all(headers)
       end
     end.flatten
   end
+
+  def self.upload_document(headers, file_name, file_body, claim_id, tracked_item_id)
```

Comment:
EVSS is back up now. In testing it seems like I can set claim id and tracked item id to anything and it will work.